packages/forma-36-react-components/src/components/TextInput/TextInput.tsx

```diff
@@ -26,6 +26,7 @@ export type TextInputProps = {
     | 'date'
     | 'time';
   name?: string;
+  labelText?: string;
```

Comment:
massive nit: personally, I'd expect a label to show up along the input if I used that `labelText` prop and not for it to just be an aria-label.

Reply:
we went with `labelText` as this fix for v3, and this how it used on TextField, in v4 however, It's renamed to `label` which resolve to `aria-label` :) but I agree, it's a bit confusing